### PR TITLE
admin sign in frontend

### LINKED ISSUE #239: Admin Sign in Page
#### Description
image

### PR DESCRIPTION
**Pull request by IshaanRSharma**
## Status:
:construction: In development

## Description :star2:
sign in admin

A sign-in page for admins.
Fixes #239 
 
## TODOs :star:
Auth connection.

### CODE REVIEW

**Review by AngelaLuo49021 — CHANGES_REQUESTED** [review]
can you pull from dev so you get the updated navbar at the top your branch v out of date

**Review by AngelaLuo49021 — APPROVED** [review]
Look good

File: package.json [hunk 1]
```diff
@@ -9,5 +9,9 @@
         "setup:backend": "cd backend && poetry install",
         "setup:frontend": "cd frontend && yarn",
         "setup": "concurrently \"yarn setup:backend\" \"yarn setup:frontend\""
+    },
+    "dependencies": {
+        "dropzone": "^5.8.1",
+        "react-dropzone": "^11.3.1"
```
Comment (kenetec) [inline package.json:15]: can you add these to the frontend's package json instead

**Review by kenetec — CHANGES_REQUESTED** [review]
looks gucci, just a quick change and ur good to go

**Review by kenetec — APPROVED** [review]
sweeet 😎

**Pull request merged**